scripts/run-typewriter-clean-metadata.ps1

```diff
@@ -7,13 +7,21 @@ Push-Location -Path $env:TypewriterDirectory
 
 & chmod +x $env:TypewriterExecutable
 
+$additionalFlags = ""
 if (Test-Path env:RemoveAnnotations) # Use the XSLT with a flag to remove capability annotations, and add documentation annotations to the metadata.
 {
-    & $env:TypewriterExecutable -v Info -m $env:InputMetadataFile -o $env:OutputPath -g $env:GenerationMode -t $env:Transform -d $env:DocsDirectory -e $env:endpointVersion -r $env:RemoveAnnotations -f $env:OutputMetadataFileName
+    $additionalFlags += " -r $env:RemoveAnnotations"
 }
-else # Use the XSLT with default transform values and add documentation annotations to the metadata.
+if(Test-Path env:AddInnerErrorDescription)
 {
-    & $env:TypewriterExecutable -v Info -m $env:InputMetadataFile -o $env:OutputPath -g $env:GenerationMode -t $env:Transform -d $env:DocsDirectory -e $env:endpointVersion
+    $additionalFlags += " -a $env:AddInnerErrorDescription"
 }
 
+if($additionalFlags -ne "")
+{
+    $additionalFlags += " -f $env:OutputMetadataFileName"
+}
+
+Invoke-Expression "$env:TypewriterExecutable -v Info -m $env:InputMetadataFile -o $env:OutputPath -g $env:GenerationMode -t $env:Transform -d $env:DocsDirectory -e $env:endpointVersion"
```

Comment:
I believe you still need to pass `$additionalFlags` to this command.

Reply:
```suggestion
Invoke-Expression "$env:TypewriterExecutable -v Info -m $env:InputMetadataFile -o $env:OutputPath -g $env:GenerationMode -t $env:Transform -d $env:DocsDirectory -e $env:endpointVersion$additionalFlags"
```
thanks for catching that. I haven't had the opportunity to test the batch of PRs in this and the metadata repo as I'm waiting for Hidi to release first.

Reply:
Just to confirm, will the expression still be evaluated without a space between the two variables?
`$env:endpointVersion$additionalFlags` vs `$env:endpointVersion $additionalFlags`

Reply:
yep, that's expected, as you can see additionalFlags starts with a blank space, so we don't have to have custom logic here